src/controllers/activegate/reconciler/statefulset/statefulset_test.go

```diff
@@ -475,3 +569,19 @@ func buildTestInstance() *dynatracev1beta1.DynaKube {
 		},
 	}
 }
+
+func buildActiveGateVolumeDirectories(readOnly bool) []string {
```

Comment:
does this logic also exist for the implementation? It seems there we always add every volume

Reply:
Right, the logic no longer exist

Reply:
Eventually logic is needed. Volumes created if needed (depend on StatsD and RO fs)